README.md

```diff
@@ -1,5 +1,49 @@
-# nsf
-This is the working repository for the NSF Site Redesign &amp; Content Discovery project. 
+# GSA OSC Web Presence
+
+## Project Description
+18F will work to understand how GSA websites and systems are currently structured, evaluate user needs and challenges, and conduct a review of GSA’s overall digital presence. 18F will then define a strategy and roadmap for how to address the most pressing challenges with the current information architecture of GSA’s websites, ultimately driving towards a simplified site structure that better resonates with users.
+
+Following the path analysis, 18F will prepare a synthesis of findings as well as preliminary recommendations identifying the next steps towards developing the requirements for FAS to acquire a solution for OSC.
+
+During this phase, 18F will provide GSA OSC with the following:
+
+- Problem definition and potential partner impact: The explicit objectives to achieve the organization’s goal(s), based upon an investigation, and the level of impact felt by the Requesting Agency or their user base.
+- Strategy, recommendations, and concept: A stated approach to achieve an outcome based on the organization’s unique position, including but not limited to its mission, context, resources, and risks.
+- When appropriate, limited prototypes, wireframes, data, or other documents generated by 18F.
+
+
+## 18F Team
+
+- T. Carter Baxter, Front End Engineer, [@tbaxter-18F](https://github.com/tbaxter-18f)
+- Ron Bronson, Service Designer (Project Lead), [@quarterback](https://github.com/quarterback)
+- Jeff Durland, Content Designer, [@durland18F](http://github.com/durland18F)
+- Austin Hernandez, Visual Designer, [@austinhernandez](http://github.com/austinhernandez)
+- Olesya Minina, Account Manager, [@ominina](https://github.com/ominina)
+- Torey Vanek, Product, [@toreyvanek1](https://github.com/toreyvanek1)
+- Scott Weber, Front End Engineer, [@line47](https://github.com/line47)
+- Fureigh, Back End Engineer/Drupal Consultant, [@fureigh](https://github.com/fureigh)
```

Comment:
@tbaxter-18f, I took the liberty of adding myself, but I recognize I haven't formally been on the team. Suggested edits are welcome. :)